src/databricks/labs/ucx/runtime.py

```diff
@@ -28,18 +28,18 @@ def crawl_tables(_: MigrationConfig):
     storage location details.
 
     The gathered metadata is then subsequently organized and documented within a designated storage entity referred to
-    as the `$inventory.tables` table. This table serves as an extensive inventory, offering a well-structured and
-    readily accessible point of reference for users, data engineers, and administrators."""
+    as the `hive_metastore.ucx.tables` table. This table serves as an extensive inventory, offering a well-structured
```

Comment:
keep the `$inventory` variable here. See

https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/install.py#L280-L281